langchain4j-core/src/main/java/dev/langchain4j/data/embedding/Embedding.java

```diff
@@ -4,6 +4,12 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Represents a dense vector embedding of a text.
+ * This class encapsulates a float array that captures the "meaning" or semantic information of the text.
+ * Texts with similar meanings will have their vectors located close to each other in the embedding space.
+ * The embeddings are typically created by embedding models. See EmbeddingModel interface.
```

Comment:
please make the link clickable :  {@link dev.langchain4j.model.embedding.EmbeddingModel}